teachers_digital_platform/js/expandable-facets.js

```diff
@@ -61,12 +62,6 @@ function initialize() {
   );
   this.transition = transition.init();
 
-  if ( contains( this.ui.content, customClasses.OPEN_DEFAULT ) ) {
```

Comment:
@rrstoll I think this deleted block of code is why you lost the class names.

Reply:
That's it.  Thanks.